examples/platform/cc13x2_26x2/project_include/OpenThreadConfig.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      Overrides to default OpenThread configuration.
+ *
+ */
+
+// TODO - enable full OpenThread integration
+// https://github.com/project-chip/connectedhomeip/issues/293
+
+#ifndef OPENTHREAD_PLATFORM_CONFIG_H
+#define OPENTHREAD_PLATFORM_CONFIG_H
+
+// Disable the Qorvo-supplied OpenThread logging facilities and use
```

Comment:
Need to update these comments?

Reply:
I had forgotten about this config header, there are so many of them this one fell through the cracks. updated